lib/active_record/connection_adapters/sqlserver_adapter.rb

```diff
@@ -159,6 +159,22 @@ def supports_in_memory_oltp?
         @version_year >= 2014
       end
 
+      def supports_insert_returning?
+        true
+      end
+
+      def supports_insert_on_duplicate_skip?
```

Comment:
These are already set to false by the abstract adapter but think it better to duplicate the methods here so that it's clear what the MSSQL adapter supports.